### PR TITLE
Add debarcer 2.2.1

### PR DESCRIPTION
**Pull request by FelixMoelder**
This adds debarcer a package for De-Barcoding and Error Correction of sequencing data containing molecular barcodes.

### CODE REVIEW

**Comment by FelixMoelder** [comment]
@bioconda-bot please fetch artifacts

File: recipes/debarcer/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    - debarcer = debarcer.debarcer:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    # importlib_metadata is needed for py<38 only, but conda does not support conditional
```
Comment (bgruening) [inline recipes/debarcer/meta.yaml:23]: mh, what is this? Conda does support conditional dependencies.

File: recipes/debarcer/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    - pysam>=0.14
+    - matplotlib-base>=3.1
+    - mistune>=0.8
+    - networkx>=1.11
+    - pygal>=2.4
+    - python >=3.6
+    - scipy>=1.0
+    - seaborn>=0.9
+    - pyyaml>=5.1
+
```
Comment (bgruening) [inline recipes/debarcer/meta.yaml:36]: can you please clean up the recipes a little bit?

File: recipes/debarcer/meta.yaml (hunk 1 above)
Comment (FelixMoelder) [reply recipes/debarcer/meta.yaml]: Good point. That was a remaining from copy&pasting a template recipe.

**Review by johanneskoester — APPROVED** [review]

**Comment by FelixMoelder** [comment]
@bioconda-bot please merge

**Pull request merged**